requirements/project-min.txt

```diff
@@ -1,3 +1,4 @@
+babel>=2.9.0
```

Comment:
This file is used to run the tests with the minimum supported version of dependencies. Therefore, we need to use `==` for every dependency to ensure a newer version is not installed for the test.

Reply:
got it, fixed thanks